R/replay-html.r

```diff
@@ -113,6 +137,15 @@ replay_html.recordedplot <- function(x, name_prefix, obj_id, ...) {
   paste0("<img src='", escape_html(path), "' alt='' width='540' height='400' />")
 }
 
+#' @export
+replay_html.knit_asis <- function(x, name_prefix, obj_id, ...) {
+  # close and open pre tags because it sometimes breaks stylesheet of htmlwidgets
+  output <- paste0("</pre><div class='examples'><div class='knit_asis'>", x, "</div></div><pre class='examples'>")
```

Comment:
By cross-reference, I mean that we've now defined how to open and close and example block in two places. If we forget, and only update one, we'll get a weird display problem that probably won't get noticed for a while.